test/collection/test_config.py

```diff
@@ -82,6 +82,22 @@ def test_basic_config():
             }
         },
     ),
+    (
+        Configure.Vectorizer.multi2vec_cohere(
+            model="embed-multilingual-v2.0",
```

Comment:
Interesting. Does this work?

Afaik 3.0 is multimodal but I don't know about 2.0 (https://docs.cohere.com/docs/multimodal-embeddings#introduction-to-multimodal-embeddings)

I wonder if it should be `embed-english-v3.0`

Reply:
same here, only config generation is tested, the values itself do not matter (besides appearing in the output)